### ISSUE TITLE
As a Security Server Administrator I would like the Security Server to not return the Jetty version that is used so that automated scanners can't use it to discover a vulnerable version

### ISSUE THREAD

**Issue by raits** [nordic-institute/X-Road#1477]
Currently, the Security Server returns the Jetty version used when a request is sent to the health check port (5588). A good practice is not to return that information so it can’t be used to scan for vulnerable endpoints.

The JIRA ticket this issue is based on can be found here: https://nordic-institute.atlassian.net/browse/XRDDEV-2217
 
Acceptance criteria:

- [ ] The Security Server health check is updated not to return the Jetty version
- [ ] Other Security Server components are checked and corrected so they don’t return the servlet container version

**Comment by ejdarrow** [comment]
Hi @raits.
Just checking in, but it looks like this issue has been resolved both on the Jira object (Marked as Done) and in the code:
`eaead277 Mikk Bachmann <mikk-erik.bachmann@nortal.com> on 2/10/23 at 6:28 AM`

I'm not sure if there's work yet to do for the github issue to be closed, but I was able to confirm locally (via docker run) at least that the security server's healthcheck, 8080, and 4000 endpoints don't return server version information in the raw headers.
Maybe this one can be closed on Github?
- EJD
